dotnet-desktop-guide/net/winforms/whats-new/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: What's new in Windows Forms
+description: Learn about what's new in Windows Forms for .NET. Windows Forms. .NET provides new features and enhancements over .NET Framework.
+ms.date: 11/05/2020
+ms.topic: conceptual
+---
+
+# What's new (Windows Forms.NET)
+
+Windows Forms for .NET 5.0 adds the following features and enhancements over .NET Framework.
+
+There are a few breaking changes you should be aware of when migrating from .NET Framework to .NET 5.0. For more information, see [Breaking changes in Windows Forms](/dotnet/core/compatibility/winforms).
+
+## Enhanced features
+
+- Microsoft UI Automation patterns work better with accessibility tools like Narrator and Jaws.
+- Improved performance and reliability.
+- Better support for high DPI resolutions such as 4k monitors.
```

Comment:
We haven't done much for "better support", so I'd be cautious mentioning this. 
We have enhanced the default VB app template in be SystemAware, to match we've done for C# in .NET Core 3.0. The better high DPI support is currently slated for .NET 6.0.